modin/engines/dask/pandas_on_dask/frame/axis_partition.py

```diff
@@ -43,13 +43,15 @@ def deploy_axis_func(
             *partitions,
             pure=False,
         )
-        if num_splits == 1:
```

Comment:
The same comment as for `deploy_func_between_two_axis_partitions`.

Reply:
Answered above.